packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -0,0 +1,298 @@
+                        data-term="{{:: 'pci_databases_general_information_name' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <span
+                                data-ng-bind="$ctrl.database.description"
+                            ></span>
+                            <button
+                                class="oui-button oui-button_ghost"
+                                data-ng-click="$ctrl.goToEditName()"
+                                data-ng-disabled="!$ctrl.database.isActive()"
+                            >
+                                <span
+                                    class="oui-icon oui-icon-pen"
+                                    aria-hidden="true"
+                                ></span>
+                            </button>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'pci_databases_general_information_status_label' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <span
+                                class="oui-badge"
+                                data-ng-class="{
+                                    'oui-badge_success': ($ctrl.database.statusGroup === 'READY'),
+                                    'oui-badge_warning': ($ctrl.database.statusGroup === 'PENDING'),
+                                    'oui-badge_error': ($ctrl.database.statusGroup === 'ERROR'),
+                                }"
+                            >
+                                <span
+                                    data-ng-bind=":: ('pci_database_common_status_' + $ctrl.database.status) | translate"
+                                ></span>
+                            </span>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'pci_databases_general_information_nodes_label' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <div>
+                                <span
+                                    data-ng-repeat="node in $ctrl.database.nodes track by node.id"
+                                    class="oui-badge pr-3 pl-1"
+                                    data-ng-class="{
+                                        'oui-badge_delivery-24-hours': (node.statusGroup === 'READY'),
+                                        'oui-badge_delivery-10-days': (node.statusGroup === 'PENDING'),
+                                        'oui-badge_limited-edition': (node.statusGroup === 'ERROR'),
+                                    }"
+                                >
+                                </span>
+                            </div>
```

Comment:
Looks like you are using this block of html in two places. Here and in databases.html. Can you reuse?

Reply:
Done. Extracted to a Node component